jobs/config.json

```diff
@@ -8307,6 +8307,27 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "ci-kubernetes-e2e-kubeadm-gce-upgrade-1-6-1-7": {
```

Comment:
As discussed on Slack, can you please create the similar job for v1.7->v1.8 while you're at it using `kubeadm-upgrade`?

Then we can do this in one PR instead of two... ;)